src/sig_stfl/sig_stfl.h

```diff
@@ -216,13 +225,14 @@ typedef struct OQS_SIG_STFL_SECRET_KEY {
 	/**
 	 * set Secret Key to internal structure Function
 	 *
-	 * @param[out] sk OQS_SIG_STFL_SECRET_KEY object
-	 * @param[in] sk_len length of the returned byte string
-	 * @param[in] sk_buf The secret key represented as OQS_SIG_STFL_SECRET_KEY object
+	 * @param[in] sk OQS_SIG_STFL_SECRET_KEY object
+	 * @param[in] key_len length of the returned byte string
+	 * @param[in] sk_buf The secret key data to populate key obj
+	 * @param[in]  context application specific data
 	 * @returns  status of the operation populated with key material none-zero length. Caller
 	 * deletes the buffer. if sk_buf is NULL the function returns the length
 	 */
-	OQS_STATUS (*deserialize_key)(OQS_SIG_STFL_SECRET_KEY *sk, const size_t sk_len, const uint8_t *sk_buf);
+	OQS_STATUS (*deserialize_key)(OQS_SIG_STFL_SECRET_KEY *sk, const size_t key_len, const uint8_t *sk_buf, void *context);
```

Comment:
I prefer the `key_len` name to be `sk_len`. What do you think?

Reply:
Done.